SS2-Project/Assets/Starstorm2/Modules/Pickups/Equipments/WhiteFlag.cs

```diff
@@ -49,16 +49,13 @@ public override bool Execute(EquipmentSlot slot)
 
         public override void Initialize()
         {
-            BuffOverlays.AddBuffOverlay(_surrenderBuff, _overlay);
+            BuffOverlays.AddBuffOverlay(SS2Content.Buffs.BuffSurrender, _overlay);
```

Comment:
90% sure SS2Content type fields aren't filled in yet during Initialize. Grab the BuffDef from the asset collection instead.